packages/@tinacms/graphql/src/types.ts

```diff
@@ -240,6 +243,7 @@ interface InnerObjectFields<WithNamespace extends boolean> extends TinaField {
 interface InnerObjectFieldsWithNamespace<WithNamespace extends boolean>
   extends TinaField {
   type: 'object'
+  required: false
```

Comment:
Should this one also be `required?: false`?

Reply:
Yes it should! good catch, thank you